From 0a93f23387877c08cabfada63480b8ce7bbce4f9 Mon Sep 17 00:00:00 2001
From: Loc Vu <loc.vu.zn@renesas.com>
Date: Mon, 9 Apr 2018 18:54:02 +0700
Subject: [PATCH 083/441] mmc: core: Add property cap-mmc-wait-busy

This patch enables an additional device tree property named
cap-mm-wait-busy to support MMC_CAP_WAIT_WHILE_BUSY.

Signed-off-by: Binh Nguyen <binh.nguyen.uw@rvc.renesas.com>
Signed-off-by: Loc Vu <loc.vu.zn@renesas.com>
---
 drivers/mmc/core/host.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/mmc/core/host.c b/drivers/mmc/core/host.c
index da950c4..0388b47 100644
--- a/drivers/mmc/core/host.c
+++ b/drivers/mmc/core/host.c
@@ -255,6 +255,8 @@ int mmc_of_parse(struct mmc_host *host)
 		host->caps |= MMC_CAP_SD_HIGHSPEED;
 	if (of_property_read_bool(np, "cap-mmc-highspeed"))
 		host->caps |= MMC_CAP_MMC_HIGHSPEED;
+	if (of_property_read_bool(np, "cap-mmc-wait-busy"))
+		host->caps |= MMC_CAP_WAIT_WHILE_BUSY;
 	if (of_property_read_bool(np, "sd-uhs-sdr12"))
 		host->caps |= MMC_CAP_UHS_SDR12;
 	if (of_property_read_bool(np, "sd-uhs-sdr25"))
-- 
2.7.4

